airbyte-cdk/java/airbyte-cdk/README.md

```diff
@@ -166,6 +166,7 @@ MavenLocal debugging steps:
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                                                        |
 |:--------|:-----------|:-----------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0.17.0  | 2024-01-31 | [\#33606](https://github.com/airbytehq/airbyte/pull/33606) | Add updated Initial and Incremental Stream State definitions for DB Sources.                                                                                   |
```

Comment:
not a breaking change per se - but a significant version update. 
Should we make it a 1.0.0 release or let it bake for a bit?